example/android/app/build.gradle

```diff
@@ -55,7 +55,7 @@ android {
 
     defaultConfig {
         applicationId "com.nimroddayan.flutternsd.example"
-        minSdkVersion 23
+        minSdkVersion 21
```

Comment:
Please keep the existing minimum

Reply:
Ok. By the way, is there a reason for this minimum?